Tools/CMake/AMReXParallelBackends.cmake

```diff
@@ -229,16 +243,16 @@ if (AMReX_HIP)
    endif()
    # AMD's or mainline clang++ with support for "-x hip"
    # Cray's CC wrapper that points to AMD's clang++ underneath
-   if(NOT ${_this_comp} STREQUAL hipcc)
+   if(NOT IS_PRE_ROCM4_HIPCC)
        target_link_libraries(amrex PUBLIC hip::device)
```

Comment:
Hm, that unwrapping to `clang++` now adds no `-x hip` flags for CXX targets and this cannot be linked until the Fortran parts are fixed.

Reply:
Maybe we can set the CXX compiler for <4.4 back to `hpcc` of `ROCMClang` is detected...